backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/window/api/IADWindowDAO.java

```diff
@@ -140,4 +140,7 @@ public interface IADWindowDAO extends ISingletonService
 	ImmutableSet<AdWindowId> retrieveAllActiveAdWindowIds();
 
 	Optional<QuickInputConfigLayout> getQuickInputConfigLayout(AdTabId adTabId);
+
+	@NonNull
+	AdWindowId retrieveOverrideWindowIdOrWindowId(@NonNull final AdWindowId windowId);
```

Comment:
pls remove this. if u really need it then pls use de.metas.document.references.zoom_into.SqlCustomizedWindowInfoMapRepository#get